frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -50,88 +58,132 @@ const FieldsAndSoilComponent: React.FC<InputModuleProps> = ({ farmDetails, updat
+    farmInfo.Fields.push({
+      FieldName: values.FieldName,
+      Area: values.Area,
+      Comments: values.Comments,
+    });
+    console.log(farmInfo);
     setFieldsInfo(farmInfo);
+    setFieldIndex((prevIndex) => prevIndex + 1);
+    setSubmitted(true);
   };
 
+  const addNewField = () => {
+    setFieldAdd(true);
+  };
+  console.log(`Field add state is ${fieldAdd}`);
+  console.log(`Field submitted state is ${isSubmitted}`);
   return (
-    <Formik
-      initialValues={initialValues}
-      validationSchema={validationSchema}
-      onSubmit={onSubmit}
-    >
-      {({ values }) => (
-        <Form>
-          <StyledFarmInfo>
-            <div id="inputContainer">
-              <CustomField
-                label="Field name"
-                id="FieldName"
-                name="FieldName"
-                type="text"
-              />
-              <CustomField
-                label="Area"
-                id="Area"
-                name="Area"
-                type="number"
-                width="20%"
-              />
-            </div>
-            <StyledTextAreaContainer>
-              <CustomTextArea
-                name="Comments"
-                id="Comments"
-                label="Comments (optional)"
-                placeholder="e.g., poor drainage in southwest corner (no need to specify crop here)"
-                width="50%"
-              />
-              <StyledButtonGroupContainer>
-                <Button
-                  type="reset"
-                  size="sm"
-                  disabled={false}
-                  text={ComponentText.CANCEL}
-                />
-                <Button
-                  type="button"
-                  size="sm"
-                  disabled={false}
-                  text={ComponentText.ADD}
-                  handleClick={() => addFarmInfo(values)}
-                />
-              </StyledButtonGroupContainer>
-            </StyledTextAreaContainer>
-            <StyledListContainer>
-              <StyledListItem>
-                <h4>Field Name</h4>
-                <p>{fieldsInfo.FieldName}</p>
-              </StyledListItem>
-              <StyledListItem>
-                <h4>Area</h4>
-                <p>{fieldsInfo.Area}</p>
-              </StyledListItem>
-              <StyledListItem>
-                <h4>Comments</h4>
-                <p>{fieldsInfo.Comments}</p>
+    <>
+      {isSubmitted ? (
+        <StyledFieldInfoList>
+          <StyledListContainer>
+            <StyledListItem width="20%">
+              <h4>Field Name</h4>
+              <p>{fieldsInfo.Fields[fieldIndex].FieldName}</p>
+            </StyledListItem>
+            <StyledListItem width="20%">
+              <h4>Area</h4>
+              <p>{fieldsInfo.Fields[fieldIndex].Area}</p>
+            </StyledListItem>
+            <StyledCommentContainerDesktop>
+              <StyledListItem width="100%">
+                <h4>Comments (optional)</h4>
```

Comment:
Improper use of header elements.
Header elements have semantic meaning. Please refer to this article https://blog.hubspot.com/marketing/header-tags, 

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/62873746/fd651c6d-0827-4bc4-ae11-c2420e8dacfc)

Please consider [HeadingsMap](https://chromewebstore.google.com/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) to look over your work